util/opentitan-pgm-fpga/opentitan-pgm-fpga

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Program an FPGA with a bitstream
+"""
+import argparse
+import os
+import subprocess
+import sys
+
+
+def program_bitstream_with_vivado(part,
+                                  bitstream,
+                                  hw_target=None,
+                                  jtag_freq=None):
```

Comment:
I still don't think these need a default argument: there's exactly one call site, and it passes all four arguments.